frontend/apps/reader/modules/readertoc.lua

```diff
@@ -1051,6 +1053,16 @@ Enabling this option will restrict display to the chapter titles of progress bar
             UIManager:show(items_font)
         end,
     }
+    menu_items.toc_items_show_separator = {
+        text = _("Show ToC entries separator"),
```

Comment:
`Add a separator between ToC entries`?

Reply:
https://github.com/koreader/koreader/blob/ea3fa5c2c7ed7330e8e1e6ef838b780e7998ea51/frontend/apps/reader/modules/readerfooter.lua#L999

Reply:
That's a single separator above a single pane, not exactly comparable ;).